app/models/redde/photo.rb

```diff
@@ -0,0 +1,16 @@
+class Redde::Photo < ActiveRecord::Base
+  mount_uploader :src, PhotoUploader
+  self.table_name = 'redde_photos'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.